test/java/src/test/java/TestArticle.java

```diff
@@ -0,0 +1,36 @@
+import static org.junit.Assert.*;
+import java.io.IOException;
+import org.junit.Test;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+
+import io.sysl.issue66.Article;
+import io.sysl.issue66.ArticleModelJsonDeserializer;
+import io.sysl.issue66.Content;
+
+public class TestArticle
+{
+    @Test
+    public void testJson() throws IOException {
+        Article a1 = buildArticle();
+        Article a2 = articleFromJson();
+        assertEquals(a1, a2);
+    }
+
+    public static Article articleFromJson() throws IOException {
+		JsonFactory factory = new JsonFactory();
```

Comment:
The indenting has gone funny in thus method - tabs vs spaces?

Reply:
Done, thanks. I run into this quite regularly - I think it's when I copypasta from somewhere else. Do you have a tool that you run to avoid such issues or should I just see if I can change something in my editor?